public/app/features/explore/QueryLibrary/utils/dataFetching.ts

```diff
@@ -0,0 +1,106 @@
+import { compact, uniq } from 'lodash';
```

Comment:
Moved some data fetching here for the sake of making them easier to mock in tests and make it bit more clear what is dependant on what in the queryTemplatesList component